analyzer/flow_client.go

```diff
@@ -35,12 +35,14 @@ import (
 	"github.com/skydive-project/skydive/logging"
 )
 
+// FlowClientPool describe a flow clients pool via websocket
```

Comment:
just : FlowClientPool describes a flow client pool.

Websocket is only used to get connected/disconnected event being able to send flow to an active server